geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py

```diff
@@ -289,6 +289,12 @@ def step0(self, step):
                         required, required, TEMPLATE_EXAMPLE.get('required', '')
                     )
                 )
+        if self.project.get("managed_files") is None:
+            messages.append(
+                "The element 'managed_files' is missing in the file 'project.yaml.mako', "
+                "you must for fill it with a list of regular expresion, and at list an empty array. "
```

Comment:
you must define this element with a list of regular expressions or with an empty array.

Reply:
thanks :-)